config/config.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Configuration creates a Global Conifguration to be passed throughout packages
+var (
+	Configuration *Config
+	RodeAPIHost   string
+)
+
+// Config struct for collector config
+type Config struct {
+	RodeAPI struct {
+		Address string `yaml:"address"`
+	} `yaml:"rode-api"`
+}
+
+// NewConfig creates a config structure
+func NewConfig(configPath string) (*Config, error) {
+	config := &Config{}
+
+	// Open config file
+	file, err := os.Open(configPath)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	// Init new YAML decode
+	d := yaml.NewDecoder(file)
+
+	// Start YAML decoding from file
+	if err := d.Decode(&config); err != nil {
+		return nil, err
+	}
+
+	return config, nil
+}
+
+func init() {
```

Comment:
I think I would have expected a function like this to explicitly be invoked from the `main` package, rather than config being parsed as a side effect of importing the `config` package.  Can we make this a bit more explicit?

Also, where did we land on needing a config yaml for this?  This seems like too simple of a program to require a yaml file for configuration in my opinion.

Reply:
We said we don't need a config.yaml in packages these small. We will be using flags.
I just kept the config.yaml reading logic in there for reference, but I can remove it.

With regards to the second package for managing configurations, (incoming dumb Go question), how do i pass a variable (ie the url) to a handle function?